infoschema/tables.go

```diff
@@ -680,6 +685,15 @@ var tableTiDBClusterConfigCols = []columnInfo{
 	{"VALUE", mysql.TypeVarchar, 128, 0, nil, nil},
 }
 
+var tableTiDBClusterLoadInfoCols = []columnInfo{
+	{"TYPE", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"ADDRESS", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"DEVICE_TYPE", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"DEVICE_NAME", mysql.TypeVarchar, 64, 0, nil, nil},
+	{"KEY", mysql.TypeVarchar, 256, 0, nil, nil},
```

Comment:
The column names are not friendly to users.
How about rename `TYPE` to 'NODE_TYPE` and `KEY` to `LOAD_TYPE`?

Reply:
This is consistent with other cluster table, such as: `tidb_cluster_config `, `tidb_cluster_info` .....

Reply:
Actually, I think they're all improper. You can discuss it with someone else.